browse-scaler/src/cache.js

```diff
@@ -34,19 +53,19 @@ exports.cacheImage = (key, image) => {
  * @returns {Buffer<Image>} the image associated with given cache key or null if none is found
  */
 exports.getImageFromCache = async key => {
-  try {
-    const image = await getAsync(key);
-
-    // workaround for bad cache entries that have the string "null" for the value
-    if (image && image !== 'null') {
-      console.log('got image from cache');
-      return image;
-    }
-
-    console.log('image is not in cache');
-    return null;
-  } catch (err) {
-    console.error(`Could not get image from cache: ${err}`);
-    return null;
-  }
+  let client = getClient();
+  return promisify(client.get).bind(client)(key)
+    .catch(err => {
+      console.error(`Unable to retrieve image ${key} from Redis.`);
+      return null;
+    })
+    .then(image => {
+      // workaround for bad cache entries that have the string "null" for the value
+      if (image && image !== 'null') {
+        console.log(`got image ${key} from cache`);
```

Comment:
Do we want to log this all the time?

Reply:
Eh, I don't think it hurts to know what key is being added to the cache.

Reply:
While it is good to know what key is being retrieved from the cache, we don't necessarily want to log it every single time we retrieve something from the cache. This seems excessive to me and it could potentially cause performance and space issues.